DHL/JSON/66da0fe7-7a0b-4e36-b685-8f4d6b06607f.json

```diff
@@ -0,0 +1,531 @@
+                                  "AttrMtdt": [
+                                    {
+                                      "isMan": true
+                                    }
+                                  ]
+                                }
+                              ]
+                            },
+                            {
+                              "Id": "4829cfe1-fcbb-48d2-9344-d8468de1f41d",
+                              "Nm": "Greek Vat number - Radio button",
+                              "Dsc": "Greek Vat number Radio button desc",
+                              "isImg": false,
+                              "isExcl": false,
+                              "isMan": false,
+                              "Att": [
+                                {
+                                  "Id": "0da3ad86-cc40-44ff-9ad4-60780be50fc8",
+                                  "Nm": "Do you have a Greek VAT number?",
+                                  "Dsc": "For customs clearance of shipments where the goods value is greater than EUR 999, Greece customs authorities require an EORI number.",
+                                  "AttTy": "UserEntry",
+                                  "isExcl": false,
+                                  "isMan": false,
+                                  "isHidden": true,
+                                  "UICmpt": {
+                                    "Order": 2,
+                                    "CmptNm": "PORadioBtn",
+                                    "DisplayTtl": "",
+                                    "Tooltip": "",
+                                    "CmptTy": "RadioButton",
+                                    "DisplayDsc": {
+                                      "DisplayTxt": "Do you have a Greek VAT number??"
+                                    },
+                                    "ValTy": "Standard"
+                                  },
+                                  "Enum": [
+                                    {
+                                      "Val": "Yes",
+                                      "DisplayTxt": "Yes",
+                                      "SrtOrd": 1,
+                                      "Nm": "Opt1"
+                                    },
+                                    {
+                                      "Val": "No",
+                                      "DisplayTxt": "No",
+                                      "SrtOrd": 2,
+                                      "Nm": "Opt2"
+                                    }
+                                  ],
+                                  "Rul": [
+                                    {
+                                      "RulId": "55b3c76f-cf7e-4a24-abeb-57694b7cd4ef",
+                                      "Nm": "Rule condition to Greek vat number",
+                                      "Cond": "**.Att[Id='6a2f893d-8c39-42ed-9b9e-450d21092907'].Val='Opt1'",
```

Comment:
update cond to -> "Cond": "**.Att[Id='6a2f893d-8c39-42ed-9b9e-450d21092907'].Val='Opt1' and **.Att[Id='0da3ad86-cc40-44ff-9ad4-60780be50fc8'].Val='Opt1'"

Reply:
Done